MEMBER_TRAVEL_FUND.md

```diff
@@ -10,40 +10,72 @@ and the Node.js Foundation projects.
 
 ## Process
 
-Any member can apply for travel funds. Each request may take up to a month
-for the [Admin members](https://github.com/nodejs/admin#governance-and-current-members)
-to approve or decline.
+Any member can apply for travel funds.
```

Comment:
Probably not what this PR has to deal with, but it would be nice if https://github.com/nodejs/members is mentioned here and a note about the individual membership registration is added.